### ISSUE TITLE
Looks good, having trouble setting it as the theme though

### ISSUE THREAD

**Issue by rcrooks** [lukeomalley/dark-rock-theme#1]
Hi @lukeomalley the screenshots look great, and it does install but I can't set it as the color theme. Guessing the issue is in the package.json but haven't looked. I'm on a Mac if that matters.

Best,
Robert

**Comment by rcrooks** [comment]
@lukeomalley I found the issue in the package.json file:
```
"path": "./themes/Dark Rock-color-theme.json"
```
that should be:
```
"path": "./themes/dark-rock-color-theme.json"
```

**Comment by rcrooks** [comment]
I'll fix and submit a pull request if you like

**Comment by lukeomalley** [comment]
Hey @rcrooks thanks for bringing this to my attention! I just published a new version of the theme (v1.0.2) with the fix. Try it again and let me know if you still have any issues.

**Comment by rcrooks** [comment]
yes, that fixed it - thanks!
